File: 프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.js
function solution(my_string) {
    // my_string에서 숫자만 추출하고 배열로 변환
    let arr = my_string.replace(/[^0-9]/g, "").split("");
    // 숫자 배열을 정렬
    arr.sort();
    // 정수 배열로 변환
    let answer = arr.map(num => parseInt(num));
    return answer;
}

File: 프로그래머스/lv0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.java
import java.util.Arrays;

class Solution {
    public int[] solution(String my_string) {
        String[] arr = my_string.replaceAll("[^0-9]", "").split("");
        // my_string에 숫자만 남긴 후 arr로 변환한다
        Arrays.sort(arr);
        // 정렬한다
        int[] answer = new int[arr.length];
        
        for(int i = 0; i < answer.length; i++) {
            answer[i] = Integer.parseInt(arr[i]);
        }   // arr의 원소를 형변환하여 넣어준다
        return answer;
    }
}